mapsforge-core/src/main/java/org/mapsforge/core/mapelements/PointTextContainer.java

```diff
@@ -64,23 +64,23 @@ protected PointTextContainer(Point point, double horizontalOffset, double vertic
 
     @Override
     public boolean clashesWith(MapElementContainer other, Rotation rotation) {
-        if (super.clashesWith(other, rotation)) {
-            return true;
```

Comment:
Isn't `super.clashesWith` needed anymore?

Reply:
> Isn't `super.clashesWith` needed anymore?

No, `super.clashesWith` does not take rotation into account, so it is not relevant for labels. The important part of the `super.clashesWith` was put here, i.e. the `Display.ALWAYS == this.display` checks.